### ISSUE TITLE
Ebib support

### ISSUE THREAD

**Issue by TheJJ** [org-roam/org-roam-bibtex#230]
I'd like to be able to create/open an associated org-roam note from an [ebib](https://joostkremers.github.io/ebib/ebib-manual.html#notes-files) entry.

**Comment by myshevchuk** [comment]
Hi. This is something I've been wanting to implement since a long time. However, it appears that Ebib's note taking system is heavily melted into Ebib core and would not be that easy to hook into it. It'll take quite some time. Any help is appreciated.

**Comment by xwzliang** [comment]
I’m not an expert on this, but I tried creating a custom function and call it in an ebib index buffer, it seems working fine:
```
(defun my-ebib-popup-note (key)
    (interactive (list (ebib--get-key-at-point)))
    (orb-edit-note key))
```

**Comment by myshevchuk** [comment]
@xwzliang, yes, well that's really simple. A few more things need to be done to get the indicator support. It seems that we just need to completely shadow several Ebib note-related functions. That's a somewhat brute force approach, but hey.

**Comment by xwzliang** [comment]
@myshevchuk Yes it’s more of a hack, but for now it suits my simple needs :)
